src/System.Diagnostics.Debug/src/System.Diagnostics.Debug.csproj

```diff
@@ -12,7 +12,9 @@
     <NoWarn>0436</NoWarn>
     <ResourcesSourceOutputDirectory Condition="'$(TargetGroup)' != ''">None</ResourcesSourceOutputDirectory>
     <!-- Suppress warnings for type conflicts between SafeFileHandle in partial facade and mscorlib -->
-    <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
+    <PackageTargetRuntime Condition="'$(TargetsWindows)' == 'true'">win7</PackageTargetRuntime> 
+    <PackageTargetRuntime Condition="'$(TargetsUnix)' == 'true'">unix</PackageTargetRuntime> 
+    <PackageTargetRuntime Condition="'$(TargetGroup)'=='netcore50aot'" />
```

Comment:
I don't think we want to do that, that would remove the default PackageTargetRuntime which is 'aot'.